app/staging/domain/case-summary.js

```diff
@@ -0,0 +1,41 @@
+class CaseSummary {
```

Comment:
If my understanding is correct, it would make more sense to call this `cases-summary` or `workload-summary` as things like communityTiers, custodyTiers etc are aggregations of multiple cases.